magefile.go

```diff
@@ -99,7 +100,8 @@ var (
 	locations                      = newLocations()
 	logger             logr.Logger = stdr.New(nil)
 	applicationVersion string
-	// Push to development registry instead of pushing to quay.io.
+	imageRegistry      string
+	// Push to development registry instead of pushing to `imageOrg`.
```

Comment:
a really small nit: shouldn't this comment mention `imageRegistry` instead of `imageOrg`? (I might be wrong here)

Reply:
Exactly